src/views/HomeView/HomeView.js

```diff
@@ -1,63 +1,56 @@
 import React, { useState, useEffect } from 'react';
-import { Route } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import './HomeView.css';
 
-// import Header from '../../components/Header/Header';
-// import Footer from '../../components/Footer/Footer';
 import UserInteractions from '../../components/UserInteractions/UserInteractions';
 import FormModal from '../../components/FormModal/FormModal';
-// import Search from '../../components/Search/Search';
 import Feed from '../../components/Feed/Feed';
-// import Post from '../../components/Post/Post';
 
 const HomeView = () => {
-
-  const [allPosts, setAllPosts] = useState([]);
-  const [userLocation, setUserLocation] = useState({});
-  const [error, setError] = useState('');
-
-  const getUserLocation = async () => {
-    // update with URL for geocoding location API
-    const url = '';
-    setError('');
-
-    try {
-      const response = await fetch(url)
-      const userLocation = await response.json()
-      setUserLocation(userLocation);
-    } catch(error) {
-      setError(error.message);
-    }
-  }
-
-  const getAllPosts = async () => {
-    // update with URL to BE to request from when avail.
-    const url = '';
-    setError('');
-
-    try {
-      const response = await fetch(url)
-      const allPosts = await response.json()
-      setAllPosts(allPosts);
-    } catch(error) {
-      setError(error.message);
-    }
-  }
+  const [isHome, setIsHome] = useState(false)
+  // const [allPosts, setAllPosts] = useState([]);
+  // const [userLocation, setUserLocation] = useState({});
+  // const [error, setError] = useState('');
+
+  // const getUserLocation = async () => {
+  //   // update with URL for geocoding location API
+  //   const url = '';
+  //   setError('');
+  //
+  //   try {
+  //     const response = await fetch(url)
+  //     const userLocation = await response.json()
+  //     setUserLocation(userLocation);
+  //   } catch(error) {
+  //     setError(error.message);
+  //   }
+  // }
+  //
+  // const getAllPosts = async () => {
+  //   // update with URL to BE to request from when avail.
+  //   const url = '';
+  //   setError('');
+  //
+  //   try {
+  //     const response = await fetch(url)
+  //     const allPosts = await response.json()
+  //     setAllPosts(allPosts);
+  //   } catch(error) {
+  //     setError(error.message);
+  //   }
+  // }
```

Comment:
All of this is lingering from what I'd started doing before stepping into the CI config process... I think since we'll have the graphQL endpoint(s) and be structuring things a bit differently, we'll probably end up losing most of this in the next PR.

Reply:
Yeah, I left this in for reference but we'll have to query a bit differently with graphQL. Looks pretty simple to add loading and error handling with Apollo as well.